### PR TITLE
Mode list in quest log

### PR DESCRIPTION
**Pull request by Jaccouille**
Could have thought of a better way to do this....

### CODE REVIEW

File: wurst/systems/modes/GameModeInit.wurst [hunk 1]
```diff
@@ -0,0 +1,335 @@
...
+public constant string array LOWER_ITEM_LIMIT       = ["lower-item-limit"      , "lil"  , "Lower item spawn rate on the map"]
+public constant string array OLD_ELEMENTALIST       = ["old-elementalist"      , "oe"   , "Replaces new elementalist with the old elementalist."]
+public constant string array DEV                    = ["dev"                   , "d"    , "Development mode (-qp, -tm, -st)"]
+public constant string array TAMEABLE_FAWN          = ["tameable-fawn"         , "tf"   , "Tame fawns with acorns"]
+
+
+init
+    GameMode.initialize()
+
+    new GameMode(PLAYER_PER_TRIBE[0], PLAYER_PER_TRIBE[1], PLAYER_PER_TRIBE[2]) (mode, args) ->
```
Comment (jonathanmclaus) [inline wurst/systems/modes/GameModeInit.wurst]: Can you not avoid unpacking each array? Could you make the `GameMode` constructor take in an array of strings instead?

**Comment by jonathanmclaus** [comment]
Why don't you just leave the current system alone and make so that the `GameMode` constructor registers the mode with a global list, then create a quest on start-up by translating the list into a quest description?

**Comment by theQuazz** [comment]
I’ll share tomorrkw

**Comment by theQuazz** [comment]
@Jaccouille here is what I'm thinking: https://github.com/island-troll-tribes/island-troll-tribes/pull/337

**Comment by Jaccouille** [comment]
Moved to #337

**Pull request closed without merging**